src/fiskaltrust.Launcher/Helpers/NumberToStringConverter.cs

```diff
@@ -0,0 +1,42 @@
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace fiskaltrust.Launcher.Helpers
+{
+    public class NumberToStringConverter : JsonConverter<string>
+    {
+        public override bool CanConvert(Type typeToConvert)
```

Comment:
Just to be sure: `typeToConvert` is the target type?

Reply:
correct, we can actually remove this override as thats just the default